lib/active_shipping/label.rb

```diff
@@ -0,0 +1,10 @@
+module ActiveShipping
+  class Label
+    attr_reader :tracking_number, :base64_img_data
+
+    def initialize(tracking_number, base64_img_data)
+      @tracking_number = tracking_number
+      @base64_img_data = base64_img_data
```

Comment:
Storing arbitrary binary in Ruby strings is totally ok, in fact it is the preferred method of storage for random binary. You can use the standard library `require "base64"` module to decode the responses into raw format.

That way it will be easier to deal with in ways like directly writing it to a file and it doesn't unnecessarily expose the implementation details of the UPS and Fedex APIs.